subscription-manager.spec

```diff
@@ -864,6 +864,13 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
     %endif
 %endif
 
+# prevent build errors on systems without %{use_subscription_manager_migration} set to true. F.e. SUSE distributions
+%if !%{use_subscription_manager_migration}
+    rm %{buildroot}%{completion_dir}/rhn-migrate-classic-to-rhsm
```

Comment:
This probably is not proper fix for your problem, because I'm getting following build error on system, where %{use_subscription_manager_migration} is false:

```
+ rm /home/jenkins/workspace/subscription-manager/subscription-manager-tito-tests-pr/tito/rpmbuild-subscription-managerx0y17jb4/BUILDROOT/subscription-manager-1.28.0-1.git.52.a116e65.fc30.x86_64/usr/share/bash-completion/completions/rhn-migrate-classic-to-rhsm
rm: cannot remove '/home/jenkins/workspace/subscription-manager/subscription-manager-tito-tests-pr/tito/rpmbuild-subscription-managerx0y17jb4/BUILDROOT/subscription-manager-1.28.0-1.git.52.a116e65.fc30.x86_64/usr/share/bash-completion/completions/rhn-migrate-classic-to-rhsm': No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.2cU8OM (%install)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.2cU8OM (%install)

```

Your proposed change does not give me much sense. Why should we remove `rhn-migrate-classic-to-rhsm` and other files from `%{buildroot}`, when these files are not going to be installed to system?

Reply:
This is odd. I actually tested the latest tag/release with the spec file from master. Maybe this causes the error as some affected files may have changed since 1.28.0-1

This is the current state where I applied my own patch to get the built working.
https://build.opensuse.org/package/show/systemsmanagement:katello/subscription-manager

I will build the latest master in a branch / fork in my home project and try to reproduce the error I got.
Thank you. You will hear from me ASAP